Duplicati/GUI/Duplicati.GUI.TrayIcon/Program.cs

```diff
@@ -174,7 +177,10 @@ public static void RealMain(string[] _args)
 
             if (options.TryGetValue(HOSTURL_OPTION, out url))
                 serverURL = new Uri(url);
-            
+
+            System.Net.ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12 | SecurityProtocolType.Tls11;
+            new Duplicati.Library.Modules.Builtin.HttpOptions().Configure(options);

```

Comment:
Yeah, you are right, I had same worries, especially with override in backup.
Same implies for backup, that this settings could influence backup options.